proto/spanner.d.ts

```diff
@@ -1146,16 +1146,19 @@ export namespace google {
                 paramTypes?: ({ [k: string]: google.spanner.v1.IType }|null);
 
                 /** ExecuteSqlRequest resumeToken */
-                resumeToken?: (Uint8Array|null);
+                resumeToken?: (Uint8Array|string|null);
```

Comment:
I assume these are auto-generated changes?

Reply:
Yes, although it could be that this change is only 'generated' because of the way I manually generated the proto changes. It should however not make any difference for this PR once the actual proto change has been merged.

Reply:
It seems that the generated code change is on master so I think you should be able to revert this file.